tests/test_features.py

```diff
@@ -10,14 +15,6 @@
 except KeyError:
     pass
 
-import matplotlib
```

Comment:
Option 3: librosa is right and I am dumb?  It's totally possible that I forgot how normalize works.

Shouldn't max-normalization along `axis=2` be equivalent to `S / S.max(axis=2, keepdims=True)`?  If so, then the first example is doing it right.

Re-reading the code (in master), everything looks as I'd expect it to according to numpy broadcast rules.